.github/workflows/build-container.yml

```diff
@@ -36,7 +36,18 @@ jobs:
           - image: 'nvidia/cuda:11.8.0-runtime-rockylinux8'
             tag_str: 'cuda_11_8_0'
         repo: ['development', 'testing', 'release']
-        series: ['3.6']
+        series: ['3.6', '23']
+        # Exclude EL7 for OSG 23 builds
+        # FIXME: exclude testing and release for OSG 23 builds
```

Comment:
uh... can you do this?

Reply:
Yup! https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs#excluding-matrix-configurations You can also do includes